test/dynamo/test_modules.py

```diff
@@ -1100,6 +1100,7 @@ def test_self_mutating1(self):
         out4 = [opt_m4(i), opt_m4(i), opt_m4(i)]
         self.assertTrue(torch._dynamo.testing.same(out2, out3))
         self.assertTrue(torch._dynamo.testing.same(out2, out4))
+        # total frame count is 6 because 3 per module
```

Comment:
This comment makes things more confusing

Reply:
This is left over. I will delete.